ravenframework/Models/ROM.py

```diff
@@ -172,6 +172,34 @@ def applyRunInfo(self, runInfo):
     """
     self.numThreads = runInfo.get('NumThreads', 1)
 
+  def _manipulateInput(self,dataIn):
+    """
+      Method that is aimed to manipulate the input in order to return a common input understandable by this class
+      @ In, dataIn, object, the object that needs to be manipulated
+      @ Out, inRun, dict, the manipulated input
+    """
+    if len(dataIn)>1:
+      self.raiseAnError(IOError,'Only one input is accepted by the model type '+self.type+' with name'+self.name)
+    inRun = super()._manipulateInput(dataIn)
+    return inRun
```

Comment:
I think you can remove these lines. I think we only need to keep one check

Reply:
Fixed